app/src/main/java/bg/zahov/app/ui/history/HistoryFragment.kt

```diff
@@ -61,42 +60,15 @@ class HistoryFragment : Fragment() {
                 }
             }
         })
-        binding.apply {
-            val historyAdapter = HistoryAdapter().apply {
-                itemClickListener = object : HistoryAdapter.ItemClickListener<Workout> {
-                    override fun onWorkoutClick(item: Workout, position: Int) {
-                        findNavController().navigate(
-                            R.id.history_to_history_info,
-                            bundleOf(workoutId to item.id)
-                        )
-                    }
-                }
-            }
-
-            workouts.apply {
-                adapter = historyAdapter
-                layoutManager = LinearLayoutManager(context)
-            }
-
-            historyViewModel.state.map { HistoryUiMapper.map(it) }.observe(viewLifecycleOwner) {
-                circularProgressIndicator.visibility = it.loadingVisibility
-                workouts.visibility = it.workoutVisibility
-                historyAdapter.updateItems(it.workouts)
-            }
-        }
     }
 
     override fun onResume() {
         super.onResume()
-        requireActivity().showTopBar()
-        requireActivity().showBottomNav()
-    }
-    override fun onDestroyView() {
-        super.onDestroyView()
-        _binding = null
+        activity?.showTopBar()
+        activity?.showBottomNav()
     }
 
     companion object {
-        const val workoutId = "WORKOUT_ID"
+        const val WORKOUTID = "WORKOUT_ID"
```

Comment:
Id can be more descriptive, e.g. `WORKOUT_ID_ARG_KEY`

Reply:
👍